workloads/kube-burner/common.sh

```diff
@@ -166,17 +166,66 @@ snappy_backup() {
  store_on_elastic
 }
 
-remove_update_taint() {
-  # This is only here until BZ https://bugzilla.redhat.com/show_bug.cgi?id=2035005 gets resolved
-  oc adm taint nodes UpdateInProgress:PreferNoSchedule- --all || true
+get_network_type() {
+if [[ $NETWORK_TYPE == "OVNKubernetes" ]]; then
+  network_ns=openshift-ovn-kubernetes
+else
+  network_ns=openshift-sdn
+fi
+echo $network_ns
 }
 
+check_metric_to_modify() {
+  export div_by=1
+  echo $config | grep -i memory
+  if [[ $? == 0 ]]; then 
+   export div_by=1048576
+  fi
+  echo $config | grep -i latency
+  if [[ $? == 0 ]]; then
+    export div_by=1000
+  fi
+}
+
+run_benchmark_comparison() {
+   if [[ -n ${ES_SERVER} ]] && [[ -n ${COMPARISON_CONFIG} ]]; then
+     log "Installing touchstone"
+     install_touchstone
+     NUM_LINES=${NUM_LINES:-10}
```

Comment:
Why do you want to only display a specific number of lines?, why not the entire comparison?

Reply:
It might be nice to put each comparison config data in a different tab/worksheet under the same sheet if we are going to be printing out the entire comparison